controller/lib/core/blower_fsm.cpp

```diff
@@ -90,92 +86,87 @@ static Duration ExpireDuration(const VentParams &params) {
 PressureControlFsm::PressureControlFsm(Time now, const VentParams &params)
     : inspire_pressure_(cmH2O(static_cast<float>(params.pip_cm_h2o))),
       expire_pressure_(cmH2O(static_cast<float>(params.peep_cm_h2o))),
-      setpoint_(expire_pressure_), start_time_(now),
-      inspire_end_(start_time_ + InspireDuration(params)),
+      start_time_(now), inspire_end_(start_time_ + InspireDuration(params)),
       expire_end_(inspire_end_ + ExpireDuration(params)) {}
 
-void PressureControlFsm::Update(Time now, const BlowerFsmInputs &inputs) {
+BlowerSystemState
+PressureControlFsm::DesiredState(Time now, const BlowerFsmInputs &inputs) {
+  bool is_end_of_breath = false;
+  if (now >= expire_end_) {
+    is_end_of_breath = true;
+  }
```

Comment:
Two other simplifications while we're here (unless you prefer to do in a separate PR).

1. Reverse the order so inspiration comes before expiration (code is written in the order it executes).
2. Get rid of the `else` (less indentation).